resources/versioning.md

```diff
@@ -0,0 +1,90 @@
+# Versioning policy
+
+This page describes the versioning rules Meilisearch SDKs follow after the release of Meilisearch engine v1.0.0. It explains how and when the Integrations Team should increase the MAJOR, MINOR, and PATCH of the versions.
+
+## 🤖 Basic rules
+
+Meilisearch SDKs releases follow the [SemVer rules](https://semver.org/), including the following basic ones:
+
+> 🔥 Given a version number MAJOR.MINOR.PATCH, increment the:
+>
+> 1. MAJOR version when you make incompatible API changes
+> 2. MINOR version when you add functionality in a backwards compatible manner
+> 3. PATCH version when you make backwards compatible bug fixes
+
+
+**Changes that MAY lead the Meilisearch users (developers) to change their code are considered API incompatibility and will make us increase the MAJOR version of Meilisearch SDK.**
+
+**In other terms, if the users MAY have to do more steps than just downloading the new Meilisearch package and running it, a new MAJOR is needed.**
+
+Examples of changes making the code break and then, involving increasing the MAJOR:
+
+- Any change related to the public API the SDK exposes like:
+    - Renaming a method name or classes
+    - Removing or changing a positional argument of a method.
+
+## Reasons why releasing a major version is needed
+
+### New runtimes adoption
+
+There is no strict policy yet regarding the minimum support of the language version. Example:
+
+Ruby SDK has a `required_ruby_version >= 2.6`, which prevents users from using the gem if they run their apps with a `Ruby 2.5`.
+
+It is often good to only allow newer versions of the runtimes because they are maintained :sparkles: and there are always new features is possible to adopt in the code and improve maintenance time.
+
+But there is a price for that. When that cut is made, the number of users who can adopt the SDK is reduced.
```

Comment:
You can turn it differently to make it more user-oriented, but still clear for the integration team, something like "we always try to avoid dropping support because we know we are aware some users could be left behind"

Reply:
Good idea, done! I just changed the message itself to make it in third person!